hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestContainerEndpoint.java

```diff
@@ -278,10 +410,41 @@ public void testGetKeysForContainer() {
     keyMetadataList = data.getKeys();
     assertEquals(1, keyMetadataList.size());
     assertEquals(3, data.getTotalCount());
+
+    // Now to check if the ContainerEndpoint also reads the File table
+    // Set up test data for FSO keys
+    setUpFSOData();
```

Comment:
@ArafatKhan2198 - Thanks for working on this patch. As discussed, pls add 2 test cases separately for testing prevKeyPrefix behavior for FSO bucket keys. Add 4 FSO keys on 4 separate paths.
1. prevKeyPrefix - empty
2. prevKeyPrefix - Pass 2nd path as prevKeyPrefix and assert for 3rd path in API response.

Reply:
Thanks for the comment @devmadhuu I have added separate UT's for the scenario where we pass a non-empty `prevKeyPrefix`